src/js/components/PodContainerSpecView.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import Icon from './Icon';
+import MarathonConfigUtil from '../utils/MarathonConfigUtil';
+
+const METHODS_TO_BIND = [
+];
+
+class PodContainerSpecView extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  getGeneralDetails() {
+    let {container} = this.props;
+    let {cpus, mem, disk} = container.resources;
+    let hash = {
+      'Name': container.name,
+      'CPUs': cpus,
+      'Memory (MiB)': mem,
+      'Disk Space (Mib)': disk,
+      'Command': MarathonConfigUtil.getCommandString(container)
+    };
+
+    return <DescriptionList hash={hash} />;
+  }
+
+  getLabelSection() {
+    let {container} = this.props;
+    if (!container.labels || !Object.keys(container.labels).length) {
+      return null;
+    }
+
+    console.log('Labels:', container.labels);
+    return (
+      <div>
+        <h5 className="inverse flush-top">Labels</h5>
+        <DescriptionList hash={container.labels} />
+      </div>
+      );
+  }
+
+  getEndpointsSection() {
+    let {container} = this.props;
+    let {id, endpoints} = container;
+    if (!endpoints || !endpoints.length) {
+      return null;
+    }
+
+    let portConfigurations = MarathonConfigUtil.getPortDefinitionGroups(
+        id, endpoints, function (content, linkTo) {
+          return <a href={linkTo} target="_blank">{content}</a>;
+        }
+      ).map(function ({hash, headline}, index) {
+        return (
+          <DescriptionList className="nested-description-list"
+            hash={hash}
```

Comment:
Can we create somewhere a `prune` function that removes keys with `null` or `undefined` values? Just to visually clean-up the empty labels:

![image](https://cloud.githubusercontent.com/assets/883486/18671450/6a0c0f9c-7f44-11e6-9b69-c4b71b353fd9.png)